web/client-ide/docker/Dockerfile

```diff
@@ -6,13 +6,16 @@ FROM docker.io/library/nginx:1.19.7
 
 # these installs are necessary to enable webdav class 2
 # https://github.com/arut/nginx-dav-ext-module
-RUN apt-get update && apt-get install -y nginx-extras libnginx-mod-http-dav-ext
+RUN set-eux; \
```

Comment:
Will this work w/o a space?

Reply:
It did, just printed out an error in the console but didn't stop:
```
Step 4/12 : RUN set-eux;     apt-get update;     apt-get install -y nginx-extras libnginx-mod-http-dav-ext;
 ---> Running in 6803894de8e4
/bin/sh: 1: set-eux: not found[7s]
```
And it didn't stop because I didn't have set -eux set! So good catch